homeassistant/components/notify/lametric.py

```diff
@@ -86,11 +89,11 @@ def send_message(self, message="", **kwargs):
             if "cycles" in data:
                 cycles = data['cycles']
             if "priority" in data:
-                if data['priority'] in ["info", "warning", "critical"]:
+                if data['priority'] in AVAILABLE_PRIORITIES:
                     priority = data['priority']
                 else:
-                    _LOGGER.warning("Priority '%s' invalid, using default "
-                                    "'%s'" % (data['priority'], priority))
+                    _LOGGER.warning("Priority %s invalid, using default %s" %
```

Comment:
Don't use the percent/modulo sign. Just add the variables after comma.
```py
_LOGGER.warning("Priority %s invalid, using default %s", data['priority'], priority)
```
https://docs.python.org/3/library/logging.html#logging.Logger.debug

Reply:
Whoops. I've moved on to `.format()` so I rarely use `%s` otherwise...